showpass/Showpass-E2E-tests
# TA-41 Ensure events can be searched for on the 'Upcoming' page

**Description:**
This spec will be verifying that after logging in end users are able to navigate to ‘Upcoming’ under the user settings and search for an event. Also, once the event got selected, an event card should show up.

Steps:
- Log in and navigate to https://beta.showpass.com/account/upcoming/
- Enter the event name into the search field
- Click search button (a magnifying glass)
- Make sure a correct event shows up and can be selected
- Click on the event
- Make sure, the event card gets properly displayed

**Jira ticket:**
https://showpass.atlassian.net/browse/TA-41

**How to test:**
Run the following test suite and ensure the tests pass
cypress/e2e/user-settings/upcoming.cy.js

**Test execution results:**

<img width="1709" alt="TA-41-passed" src="https://github.com/showpass/Showpass-E2E-tests/assets/138615449/439d0195-d715-4bcc-b341-875cf515d56b">
